### PR TITLE
feat(indexers): add API support for Orpheus to fetch size

### PR DESCRIPTION
**Pull request by zze0s**
Add API support for Orpheus to get torrent size.

## Future goal

Expose and filter by uploader.

### CODE REVIEW

File: internal/domain/indexer.go [hunk 1]
```diff
@@ -339,6 +339,7 @@ type TorrentBasic struct {
 	TorrentId string `json:"TorrentId,omitempty"`
 	InfoHash  string `json:"InfoHash"`
 	Size      string `json:"Size"`
+	Uploader  string `json:"Uploader"`
```
Comment (KyleSanderson) [inline internal/domain/indexer.go:342]: is this relevant to any other API members?

Comment (zze0s) [reply internal/domain/indexer.go:342]: Will be for RED, OPS to start with. Maybe PTP later!
